tests/Unit/CLI/ConfigTest.php

```diff
@@ -31,4 +31,28 @@ public function test_it_should_create_config(): void
         $classSetRulesExpected[] = ClassSetRules::create($classSet, ...[$rule]);
         $this->assertEquals($classSetRulesExpected, $config->getClassSetRules());
     }
+
+    public function test_it_should_create_config_with_only_one_rule_to_run(): void
+    {
+        $classSet = ClassSet::fromDir(__DIR__.'/foo');
+
+        $rule1 = Rule::allClasses()
+            ->that(new ResideInOneOfTheseNamespaces('App\Controller'))
+            ->should(new HaveNameMatching('*Controller'))
+            ->because('all controllers should be end name with Controller');
+
+        $rule2 = Rule::allClasses()
+            ->that(new ResideInOneOfTheseNamespaces('App\Service'))
+            ->should(new HaveNameMatching('*Service'))
+            ->because('all services should be end name with Service')
+            ->runOnlyThis();
+
+        $config = new Config();
+        $config->add($classSet, ...[$rule1, $rule2]);
+
+        $this->assertInstanceOf(Config::class, $config);
+
+        $classSetRulesExpected[] = ClassSetRules::create($classSet, ...[$rule2]);
```

Comment:
I suppose the spread operator of the one element array is a typo?

Reply:
I think it's correct, I copied the previous test and it seems to work fine.

Reply:
I was saying that:
`...[$rule2]` is the same as `$rule2`
but if you prefer to keep it like this for consistency with other tests it's ok, it's irrelevant
:slightly_smiling_face: